### ISSUE TITLE
Installation stalls on "Updating multilib database"

### ISSUE THREAD

**Issue by MintyX** [Antergos/Cnchi#214]
Hello,

I have had this issue using both the 2014.05.14 and 2014.26.05 64 bit isos. When trying to install Antegros in a VM the installation stalls on "Updating multilib databse". I have let it run for about five hours now to no avail. I looked up a solution which entailed enabling other repos, but that has not worked for me either. Does anyone have any insights?

**Comment by leesei** [comment]
I would like to add my findings on stalling of `pacman -Syy`:
In some environments, *cough* my workplace *cough*, `pacman -Syy` always stall at 80% of when downloading extra.db.
I have to use the HTTPS repos [here](https://www.archlinux.org/mirrorlist/all/https/) to pass that.

I noticed (from console log) Cnchi will update the repo with `rankmirror`, is there a way to skip this step? (I happened to keep `/etc/pacman.d/mirrorlist` opened and overwritten it with the HTTPS repo before the actual install.)
I see it downloads using HTTPS, which is very good.
